src/components/AccessibilityGuide/AriaLandmarks.mdx

```diff
@@ -0,0 +1,119 @@
+import { ArgTypes, Canvas, Description, Meta, Source } from "@storybook/blocks";
+
+import * as SearchBarStories from "../SearchBar/SearchBar.stories";
+import Link from "../Link/Link";
+import Banner from "../Banner/Banner";
+
+<Meta title="Accessibility Guide/ARIA Landmarks" />
+
+# ARIA Landmarks
+
+## Table of Contents
+
+- {<Link href="#Landmarks-overview" target="_self">Landmarks overview</Link>}
+- {<Link href="#banner" target="_self">banner</Link>}
+- {<Link href="#complementary" target="_self">complementary</Link>}
+- {<Link href="#contentinfo" target="_self">contentinfo</Link>}
+- {<Link href="#form" target="_self">form</Link>}
+- {<Link href="#main" target="_self">main</Link>}
+- {<Link href="#navigation" target="_self">navigation</Link>}
+- {<Link href="#region" target="_self">region</Link>}
+- {<Link href="#search" target="_self">search</Link>}
+- {<Link href="#resources" target="_self">Resources</Link>}
+
+## Landmarks overview
+
+Landmarks are a useful tool for people who use screen readers to orient themselves to important areas of a web page, and help quickly navigate to these areas within a website or application.
+
+ARIA’s 8 landmark roles are specifically meant to describe page organization and structure, providing keyboard navigation to each larger section. Each of these roles has a corresponding native HTML element:
+
+| Landmark role | HTML element        |
+| ------------- | ------------------- |
+| banner        | document `<header>` |
+| complementary | `<aside>`           |
+| contentinfo   | document `<footer>` |
+| form          | `<form>`            |
+| main          | `<main>`            |
+| navigation    | `<nav>`             |
+| region        | `<section>`         |
+| search        | `<search>`          |
+
+Landmarks can be indicated with `role="landmark"` or by using the landmark's HTML element.
+
+**Some Reservoir components already define their own landmark roles, and some landmarks must be defined by the consuming app since they could wrap many components.**
+
+### banner
+
+The `Header` is Reservoir's only component that falls under the `banner` category: an area that has global, rather than page-specific, information or actionable elements.
+For example, a website's logo and primary navigation would be expected child elements of a `banner` landmark. There should only be a single `banner` landmark per interface.
+
+Our `Header` contains other landmarks: `complementary` for temporary sitewide alerts, `navigation` for the site navigation. (The DS `Banner` is a `complementary` landmark).
```

Comment:
The DS `Banner` sentence can be its own sentence and perhaps highlight that the name matches a landmark role but only in name and not in semantic meaning.